main/resource-access.bs

```diff
@@ -67,13 +69,35 @@ if it needs to access resources requiring authentication (see [[#webid]]).
 When it receives a response with a `403` or `404` status code,
 it MAY repeat the request with different credentials.
 
+A Solid client MUST use the `Content-Type` request header in `PUT`, `POST` and `PATCH` requests. [[Source](https://github.com/solid/specification/issues/70#issuecomment-547924171)]
 
-## Linked Data Platform ## {#ldp}
+## Uniform Resource Identifier ## {#uri}
 
-Issue: Write Linked Data Platform section.
+### Required shared slash semantics ### {#uri-path-component}
 
-Draft:
-A Solid data pod MUST conform to the LDP specification [[!LDP]].
+Server and client share the slash semantics in the URI path component representing hierarchy [[!RFC3986]]. [[Source](https://github.com/solid/specification/issues/35#issuecomment-547949014)]
+
+Server MUST NOT create a URI differing only in the trailing slash with that of an existing URI. [[Source](https://github.com/solid/specification/issues/107#issuecomment-567482817)]
+
+Server MAY redirect a request with 301 from a non-existing URI to an existing URI with a matching path component ending with the same segment term. Behaviour pertaining to authorization MUST proceed this optional redirect. [[Source](https://github.com/solid/specification/issues/107#issuecomment-567454889)]
+
+## Linked Data ## {#linked-data}
+
+### Containment ### {#resource-containment}
+
+Solid has the notion of containers to represent a collection of linked resources to help with resource discovery and lifecycle management.
+
+There is a 1-1 correspondence between containment triples and relative reference within the path name hierarchy. [[Source](https://github.com/solid/specification/issues/98#issuecomment-547506617)]. It follows that all resources are discoverable from a container and that it is not possible to create orphan resources. [[Source](https://github.com/solid/specification/issues/97#issuecomment-547459396)]
+
+The representation and behaviour of containers in Solid is corresponds to LDP Basic Container and MUST be supported. [[Source](https://github.com/solid/specification/issues/47#issuecomment-561675764)]
+
+Server MUST create intermediate containers as well as corresponding containment triples derived from the URI path component of `PUT`, `POST`, and `PATCH` requests. [[Source](https://github.com/solid/specification/issues/68#issuecomment-561690124)]
+
+### Persistence ### {#uri-persistence}
+Servers should not re-use URIs, regardless of the mechanism by which resources are created. Certain specific cases exist where URIs may be reinstated when it identifies the same resource, but only when consistent with Web architecture's <cite><a href="https://www.w3.org/TR/webarch/#URI-persistence">URI persistence</a></cite> [[!WEBARCH]]. [[Source](https://github.com/solid/specification/issues/46#issuecomment-589619372)]
```

Comment:
1) I think there is no technical way to enforce this. It's a non-computable problem.

2) If there is, then we should write down that technical way instead of the above explanation. 

Also SHOULD NOT in caps.

Reply:
Counter-example to one possible interpretation in which resources cannot be recreated after they have been deleted: a specific ACL (which is also a resource) might be deleted and be needed again at a later time.

Reply:
https://github.com/solid/specification/issues/46#issuecomment-589619372 was agreed as non-normative. It was derived from LDP's notable information. The general case doesn't need to be "computable". Note also "we don't specify how a server or a URI owner determines representation consistency (re AWWW's good practice)" in https://github.com/solid/specification/issues/46#issuecomment-602511328. The specific case(s) eg. 401, are intended to be revisited after ~FPWD. If you still object, can we capture that in the issue?

Where the practice is applicable the text is about reinstating resources and their continuity as per URI persistence. Inability to reinstate is not covered.

Reply:
Non-normative is fine, but shall we then mark as such?

Reply:
I was reserving uppercase keywords for normative but we can also lead the text with non-normative heading.